README.md

```diff
@@ -136,7 +136,7 @@ use [pip-compile]. After modifying the file `requirements.in`, you must run the
 following command to update `requirements.txt`:
 
 ```
-script/pip-compile requirements/requirements.in
+script/pip-compile requirements.in requirements.txt
```

Comment:
Should this command be:
```
script/pip-compile requirements/requirements.in requirements/requirements.txt
```

Reply:
Given that there are several requirements files, my idea was to provide generic instructions. But it is true that it is not clear enough. I tries to fix it in 2b7cfb6. PTAL.